### DISCUSSION TITLE
The History of Naming, Zod

### DISCUSSION THREAD

**Discussion by thehouseofel** [colinhacks/zod#2252] [Q&A]
We formally request an inquiry into the origin of this project's use of the name, Zod. This name carries with it a troubled past with our people.

Is there a story behind why this name was chosen?

Respectfully
- _House of El_

**Comment by JacobWeisenburger** [comment]
![ENXX2DGXUAQkPRG](https://user-images.githubusercontent.com/31667350/228049873-f625ff6a-6934-41e2-b9e2-205052fb0149.jpg)
